libevmasm/ConstantOptimiser.cpp

```diff
@@ -93,15 +93,8 @@ bigint ConstantOptimisationMethod::simpleRunGas(AssemblyItems const& _items)
 
 bigint ConstantOptimisationMethod::dataGas(bytes const& _data) const
 {
-	if (m_params.isCreation)
```

Comment:
Why did you remove this?

Reply:
It has been moved into another function, actually those 2 commits don't need to be part of this PR as it doesn't need to use `dataGas` at all.